tests/rule_based_profiler/parameter_builder/test_value_set_multi_batch_parameter_builder.py

```diff
@@ -80,7 +80,7 @@ def test_value_set_multi_batch_parameter_builder_alice_single_batch_numeric(
     )
 
     fully_qualified_parameter_name_for_value: str = "$parameter.my_event_type_value_set"
-    expected_value_set: Set[int] = {73, 19, 22}
+    expected_value_set: List[int] = [19, 22, 73]
```

Comment:
If possible I'd love to leave this as a set which is more meaningful since this is the set of unique values.

Reply:
@anthonyburdi From our discussion -- "set" is not serializable.